### PR TITLE
[GUI] Resolves crashing when downloading

### PR DESCRIPTION
**Pull request by dongle-the-gadget**
Resolves crashing when downloading due to UI update calls being made by a non-UI thread.

### CODE REVIEW

**Comment by gus33000** [comment]
Thanks again for the PR, and sorry for the delay, I've been setting aside this project a bit too much lately ^^

Going to review it now

File: src/Applications/UUPMediaCreator/Pages/DownloadPage.xaml.cs [hunk 1]
```diff
@@ -44,7 +45,10 @@ private async void DownloadPage_Loaded(object sender, RoutedEventArgs e)
 
             App.ConversionPlan.TmpOutputFolder = await UpdateUtils.ProcessUpdateAsync(App.ConversionPlan.UpdateData, tmp.Path, App.ConversionPlan.MachineType, this, App.ConversionPlan.Language, App.ConversionPlan.Edition, UseAutomaticDownloadFolder: false).ConfigureAwait(false);
 
-            Frame.Navigate(typeof(BuildingISOPage));
+            await Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
```
Comment (gus33000) [inline src/Applications/UUPMediaCreator/Pages/DownloadPage.xaml.cs:48]: Is this needed here? This code in the UI thread, the code that doesn't is in the report function indeed. I would keep the dispatcher for the report function but I would remove it here.

**Review by gus33000 — CHANGES_REQUESTED** [review]
Just a tiny thing to change, but aside from that it is good 👍

File: src/Applications/UUPMediaCreator/Pages/DownloadPage.xaml.cs (hunk 1 above)
Comment (superkid200) [reply src/Applications/UUPMediaCreator/Pages/DownloadPage.xaml.cs:48]: Yeah, for some reason it threw a wrong thread exception, so I added that in.

**Review by gus33000 — APPROVED** [review]

**Pull request merged**